sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsActionResult.java

```diff
@@ -12,7 +13,7 @@
 /**
  * The {@link TextAnalyticsActionResult} model.
  */
-public class TextAnalyticsActionResult {
+public abstract class TextAnalyticsActionResult {
```

Comment:
Has this class been GA'd? If so this is a breaking change to make the class abstract.

Reply:
Noop. It is still in beta.